llms/generatecontent.go

```diff
@@ -17,7 +17,10 @@ type MessageContent struct {
 }
 
 // TextPart creates TextContent from a given string.
-func TextPart(s string) TextContent {
```

Comment:
We can't break the published api in this manner, this would need to be a variadic option list instead.